sympy/solvers/ode.py

```diff
@@ -4055,7 +4055,7 @@ def ode_nth_order_reducible(eq, func, order, match):
     >>> from sympy import Function, dsolve, Eq
     >>> from sympy.abc import x
     >>> f = Function('f')
-    >>> eq = Eq(x*f(x).diff(x)**2 + f(x).diff(x, 2))
+    >>> eq = Eq(x*f(x).diff(x)**2 + f(x).diff(x, 2), 0)
```

Comment:
How did this get missed before? This should have been generating a warning which should have been turned into an error by the test runner. Maybe warnings don't raise in the doctests...

Reply:
It has been raising errors in the travis tests. That's how I saw it.